.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  root: true,
+  extends: "eslint:recommended",
```

Comment:
Can we please consider conforming to the ESLint rules established in other Ably repositories, ideally AirBnB per [ably-common](https://github.com/ably/ably-common/blob/main/.eslintrc.js) and [the backend](https://github.com/ably/asset-tracking-backend-demo/blob/main/functions/.eslintrc.js).